pdagent/constants.py

```diff
@@ -49,3 +49,6 @@
 
 # PD heartbeat end-point.
 HEARTBEAT_URI = "https://api.pagerduty.com/agent/2014-03-14/heartbeat"
+
+# Maximum length of the description field (any longer and API will send 400)
+MAX_DESCRIPTION_LEN = 1024
```

Comment:
Where is this used?

Reply:
@Deconstrained Maybe we can use this combined with https://github.com/PagerDuty/pdagent/pull/111?

Reply:
@rduffield I agree 100%

I submitted this to be combined with [pdagent-integrations pull 28](https://github.com/PagerDuty/pdagent-integrations/pull/28), but simply referencing the constant in `pdagentutil._build_event_json_str` would be easier and safer and require a new release only of pdagent versus both pdagent and pdagent-integrations.